ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -197,6 +197,10 @@
   }
 }
 
+.ddp-required-question-prompt:after {
+    content: ' *';
+}
+
 .ddp-helper {
   display: none;
```

Comment:
I think this must be an existing issue, but this gets overwritten in a few places.  If you wouldn't mind changing `display: none;` to `display none !important;`, that would be helpful.

Reply:
Done! (1b014bfd) I don't have context on where this CSS class comes into play, so I didn't test it.